loader.py

```diff
@@ -221,50 +226,58 @@ def save_summary(rate, replicateId, burnIn):
             writer.writerow(data)
 
 
-def main(studyId, burnIn, subset):
+def main(configuration, studyId, burnIn, subset):
 
+    # Load the configuration
+    cfg = load_configuration(configuration)
     # Get the studies
     print("Querying for studies...")
     studies = get_studies(studyId)
     if len(studies) == 0:
         print("No studies to process!")
         return
-    
+
     # Let the user know what is going on
     if len(studies) == 1:
         print("Processing study...")
     else:
         print("Processing {} studies...".format(len(studies)))
-    
+
     counter = 1
     for study in studies:
         # Process the replicates for this study
         replicates = get_replicates(study[0], study[1])
         process_frequencies(replicates, subset)
         process_summaries(replicates, burnIn)
-        
+
         # Update the status for the user
         if len(studies) > 1:
             print("{} of {} studies complete".format(counter, len(studies)))
             counter += 1
 
 if __name__ == '__main__':
+
+
     if len(sys.argv) != 3:
-        print ("Usage: ./loader.py [studyId] [startDay]")
-        print ("studyId  - database id of the study")
-        print ("startDay - the first model day to start processing data for")
+        print("Usage: ./loader.py [configuration] [studyId] [startDay]")
+        print("configuration - the configuration file to be loaded")
+        print("studyId  - database id of the study")
+        print("startDay - the first model day to start processing data for")
         exit(0)
 
     # Prepare the environment
-    if not os.path.exists("out"): 
+    if not os.path.exists("out"):
         os.makedirs("out")
     else:
-        for filename in glob.glob("out/*summary*.csv"): os.remove(filename)
+        for filename in glob.glob("out/*summary*.csv"):
+            os.remove(filename)
 
     # Parse the parameters
+    configuration = sys.argv[1]
     studyId = int(sys.argv[1])
```

Comment:
Make sure you are testing the scripts, both of these variables are referring to the same index number.